ha/cli/exec/supportBundleExecutor.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import argparse
+import os
+
+from ha import const
+from ha.cli.exec.commandExecutor import CommandExecutor
+from cortx.utils.schema.payload import Tar
+from cortx.utils.process import SimpleProcess
+from ha.core.error import SupportBundleError
+from cortx.utils.log import Log
+
+class CreateExecutor(CommandExecutor):
+
+    def __init__(self):
+        """
+        This Class generates the Support Bundle for Component HA.
+        This method will generate bundle for HA and include the logs in it.
+        """
+        # TODO Add setup detection logic
+        self._id = None
+        self._path = None
+        self._args = None
+
+
+    def parse_cluster_args(self) -> bool:
+        '''
+           Parses the command line args.
+           Return: argparse
+        '''
+        parser = argparse.ArgumentParser(prog='support_bundle create')
+        parser.add_argument("support_bundle", help="Module name")
+        parser.add_argument("create", help="Operation")
+        parser.add_argument('--id', help='Bundle id', required=True)
+        parser.add_argument('--path', help='Bundle path', required=True)
+        self._args = parser.parse_args()
+        self._id = self._args.id
+        self._path = self._args.path
+        return True
```

Comment:
`return True` looks redundant. Is it required by some interface?

Reply:
Return value checks added to the caller